documentation/upgrading_to_bolt_3.md

```diff
@@ -0,0 +1,85 @@
+# Upgrading to Bolt 3.0
+
+Welcome to Bolt 3.0! 
+
+**Before you upgrade:**
+- Take a look at the
+  [Changelog](https://github.com/puppetlabs/bolt/blob/main/CHANGELOG.md).
+
+You can view upgrade instructions for your operating system at [Installing
+Bolt](bolt_installing.md).
+
+## Migrating configuration files
+
+Bolt 3.0 makes a lot of changes to Bolt's configuration file layout and content, including
```

Comment:
```suggestion
Bolt 3.0 makes changes to Bolt's configuration file layout and content, including
```

I think 'a lot' can be intimidating and suggests more work for the user than there might be.

Reply:
Good point! I actually spent a morning trying to configure vim to highlight a bunch of words and phrases like this ('very', 'really', etc.) and didn't have success - I might try again over the long weekend.